demo/site/src/news/blocks/NewsListBlock.loader.ts

```diff
@@ -0,0 +1,38 @@
+import { BlockLoader, gql } from "@comet/cms-site";
+import { NewsListBlockData } from "@src/blocks.generated";
+
+import { GQLNewsListBlockNewsFragment, GQLNewsListBlockQuery, GQLNewsListBlockQueryVariables } from "./NewsListBlock.loader.generated";
+
+export type LoadedData = GQLNewsListBlockNewsFragment[];
+
+export const loader: BlockLoader<NewsListBlockData> = async ({ blockData, graphQLFetch }): Promise<LoadedData> => {
+    const newsList: LoadedData = [];
+
+    // TODO create a require that allows loading multiple news by IDs at once?
```

Comment:
```suggestion
    // TODO create a resolver that allows loading multiple news by IDs at once?
```

Reply:
@nsams what do you think? Should I create a resolver for this?

Reply:
Yes, but is this something we want? Then I'd add it. Otherwise I'd remove the TODO altogether

Reply:
> Yes, but is this something we want?

in a real application: yes. But this is just a made up demo.... Maybe you could show best practice by implementing it... But if not it's also ok.

Reply:
I've added the resolver as it's a good example for how to extend the API Generator: 184a3a604469db67fc27016ae21cd3506cfaf7a0